packages/dcos-integration-test/extra/integration_test.py

```diff
@@ -1773,6 +1781,13 @@ def test_3dt_bundle_download_and_extract(cluster):
                 assert 'ip' in health_report
                 assert health_report['ip'] == slave_ip
 
+                # make sure systemd unit output is correct and does not contain error message
+                gzipped_unit_output = z.open(agent_folder + 'dcos-mesos-slave.service.gz')
+                unit_output = gzip.decompress(gzipped_unit_output.read())
```

Comment:
we should also not call .read(), potentially the logs can be very huge. Lucky we are able to read N bytes from the beginning, this should be good enough.